lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml

```diff
@@ -42,6 +45,19 @@ click_silence_alert:
   params:
     button_text: Silence Alert
   action: click_button_text
+uncheck_start_from_now:
+  element:
+    selector:
+      xpath: //span[text()='Duration']/ancestor::div[@class='co-m-pane__body-group']//input[@type='checkbox']
+    op: click
+ensure_start_from_now_checked:
+  element:
+    selector:
+      xpath: //span[text()='Duration']/ancestor::div[@class='co-m-pane__body-group']//input[@type='checkbox'][@checked]
+check_regular_expression_link:
+  element:
+    selector:
+      xpath: //a[@href='https://github.com/google/re2/wiki/Syntax']
```

Comment:
why the xpath is that?

Reply:
here I want to check the value of attribute href is what I want.